phpunit.xml.dist

```diff
@@ -6,7 +6,7 @@
 		 xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.1/phpunit.xsd"
 >
 	<php>
-		<env name="SYMFONY_PHPUNIT_VERSION" value="6.5" />
+		<env name="SYMFONY_PHPUNIT_VERSION" value="7.0" />
```

Comment:
I see there is PHPUnit 7.5. Can we use it, insted of 7.0?
Link: https://packagist.org/packages/phpunit/phpunit#7.5.16

Reply:
I updated the merge request